src/lib/crypto/plonkish_prelude/vector.mli

```diff
@@ -81,6 +81,9 @@ end
 
 (** {2 Vectors} *)
 
+(** Vector of size 32 *)
```

Comment:
I believe you could make these functors. BTW maybe we already have dependently typed vector parametrized by a Nat type?